repo-projects/windowsdesktop.proj

```diff
@@ -3,6 +3,9 @@
   <PropertyGroup>
     <!-- May stabilize. -->
     <AllowDotNetFinalVersionKindOverride>true</AllowDotNetFinalVersionKindOverride>
+
+    <BuildArgs>$(BuildArgs) /p:TargetRuntimeIdentifier=$(TargetRid)</BuildArgs>
```

Comment:
Why is this required? I don't see `TargetRuntimeIdentifier` used anywhere in windowsdesktop or arcade.